EDITORS.rst

```diff
@@ -0,0 +1,35 @@
+Welcome
+=======
+
+Thank you for helping edit the NixOS "Weekly" Newsletter!
+
+Creating a new post
+===================
+
+
+1. Copy `template.rst <./template.rst>`_ to ``content/<year>/<number>-<title>.rst``
+
+2. Make changes
+
+   Running ``nix-shell --command "pelican -r"`` will rebuild the issue
+   whenever new content is discovered. Check the ``drafts`` folder for
+   the output.
+
+   Alternatively, ``nix-build`` will build and produce ``result``.
+
+3. Create a pull-request against ``master``.
+
+   You'll be able to view the staging site at http://weekly.nixos.org/preview.
+
+4. Ask for contributions.
+
+   Some good places to do so may include:
+
+   - the mailing list
+   - social media
+   - friends/coworkers
+
+   Historically, the pull-request has been where contributions have
+   been made.
+
+5. Publish by merging into ``production``.
```

Comment:
Before publishing it would be good to write how/where to preview it online. Maybe even a hint how to debug it (eg. currently there is a bug in `.travis.yml` that merge to production fails).

Reply:
Also after this sections we need to send a newsletter and we are currently using mailchimp. I guess the easiest to explain would be to create screenshots guide or something like that.

Reply:
I added a Troubleshooting section to cover this sort of thing and am inclined to create a ticket as well.